devel/ci/cico.pipeline

```diff
@@ -47,7 +47,7 @@ def configure_node = {
     onmyduffynode 'systemctl start docker'
     // To run the integration testsuite
     onmyduffynode 'python3.6 -m ensurepip'
-    onmyduffynode 'python3.6 -m pip install \'pytest<4.0\' pytest-cov conu'
+    onmyduffynode 'python3.6 -m pip install \'pytest<4.0\' pytest-cov conu munch psycopg2'
```

Comment:
It might be ideal to run the tests in a container too so we can ```dnf install``` these.

Reply:
Same question as above: can containers run other containers?

Reply:
I don't know, but I'm comfortable with the pip installs for now.